stubs/pywin32/@tests/stubtest_allowlist.txt

```diff
@@ -2,22 +2,26 @@
 _win32typing
 win32com(ext)?.mapi._exchdapi
 win32._wincerapi
+
 # PyWin tool / debugger
 pythonwin.pywin.*
 win32com.client.combrowse
 win32com.client.tlbrowse
+
 # Utilities to generate python bindings
 win32com.client.build
 win32com.client.CLSIDToClass
 win32com.client.connect
-# Necessary for mypy to not throw AssertionError
+# dynamic.CDipatch Necessary for mypy to not throw AssertionError
```

Comment:
copy-paste error?

Reply:
I'm just being more explicit. `N` should be lowercase and I could wrap the class name in quotes.